OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/EdxRatingBar.java

```diff
@@ -0,0 +1,47 @@
+package org.edx.mobile.view.custom;
+
+import android.content.Context;
+import android.graphics.PorterDuff;
+import android.graphics.drawable.LayerDrawable;
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+import android.util.AttributeSet;
+import android.widget.RatingBar;
+
+import org.edx.mobile.R;
+
+public class EdxRatingBar extends RatingBar {
+    private final int UNSELECTED_STAR_COLOR = getResources().getColor(R.color.edx_brand_gray_back);
+    private final int SELECTED_STAR_COLOR = getResources().getColor(R.color.edx_brand_primary_base);
+    private final int SELECTED_STAR_COLOR_DARK = getResources().getColor(R.color.edx_brand_primary_dark);
+
+    public EdxRatingBar(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    public EdxRatingBar(Context context, AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+
+    public EdxRatingBar(Context context, AttributeSet attrs) {
```

Comment:
`EdxRatingBar`class needs at least one overriden constructor. We can override one and remove the rest.